process_jartic_archives.py

```
@@ -0,0 +1,280 @@
+            args.max_locations
+        )
+    )
+    
+    # Analyze if requested
+    if args.analyze and total_measurements > 0:
+        print("\nANALYZING DATA...")
+        analyze_dataset(str(output_path))
+
+
+if __name__ == "__main__":
+    main()
```

Pylint (3.3.7)

[convention] Final newline missing

(C0304)